### PR TITLE
vendor: update resumable dependency

### PR DESCRIPTION
**Pull request by stevvooe**
Updates resumable hash implementation to Go 1.8 equivalent. This should
be a major speedup, since it includes a number of optimizations from Go
1.7.

Signed-off-by: Stephen J Day <stephen.day@docker.com>

cc @dmcgowan @jlhawn @dmp42

### CODE REVIEW

**Comment by stevvooe** [comment]
Looks like this needs Go 1.8 to compile.

**Comment by dmcgowan** [comment]
This can be rebased now and run on go 1.8

**Comment by stevvooe** [comment]
@dmcgowan Done.

File: vendor.conf [hunk 1]
```diff
@@ -23,8 +23,8 @@ github.com/mitchellh/mapstructure 482a9fd5fa83e8c4e7817413b80f3eb8feec03ef
 github.com/ncw/swift b964f2ca856aac39885e258ad25aec08d5f64ee6
 github.com/spf13/cobra 312092086bed4968099259622145a0c9ae280064
 github.com/spf13/pflag 5644820622454e71517561946e3d94b9f9db6842
-github.com/stevvooe/resumable 51ad44105773cafcbe91927f70ac68e1bf78f8b4
-github.com/xenolf/lego a9d8cec0e6563575e5868a005359ac97911b5985
+github.com/stevvooe/resumable 2aaf90b2ceea5072cb503ef2a620b08ff3119870
+github.com/xenolf/lego/acme a9d8cec0e6563575e5868a005359ac97911b5985
```
Comment (dmcgowan) [inline vendor.conf]: This is incorrect, should remain as `github.com/xenolf/lego`

Comment (stevvooe) [inline vendor.conf]: Fixed.

**Comment by dmcgowan** [comment]
LGTM

**Comment by marcusmartins** [comment]
LGTM

**Pull request merged**